src/pages/user/index.tsx

```diff
@@ -25,33 +24,45 @@ const GithubConnectButton: React.FunctionComponent<{
 type ViewerAccount = {
   stripe_customer_id: string
   slug: string
+  subscriptions: any[]
+}
+
+function getAccountWithSubscription(accounts: ViewerAccount[]) {
+  // prefer the account with a subscription, otherwise grab the first, or just
+  // an empty object (which would be an error state, but possible)
+  return (
+    find<ViewerAccount>(
+      accounts,
+      (account: ViewerAccount) => account.subscriptions?.length > 0,
+    ) ||
+    first<ViewerAccount>(accounts) || {slug: ''}
+  )
 }
```

Comment:
Should we also filter/find based on the `subscription.quantity`? My assumption is that the _Account_ page is for viewing individual subscriptions, in which case we'd only want to grab an account that has a subscription with a quantity (seat) of `1`.

I'm also curious to know how we are ending up with users that have multiple accounts. Is it Bad Data™ or is it a valid scenario that we need to better account for? Maybe some data migration scenario led to a user having multiple accounts 🤔